engines/wasmtime/wasmtime.go

```diff
@@ -108,6 +124,11 @@ func (e *engine) New(_ context.Context, host wapc.HostCallHandler, guest []byte,
 	}, nil
 }
 
+// FuelConsumed returns the amount of fuel consumed in the underlying store or false if fuel consumption was not configured.
+func (i *Instance) FuelConsumed(context.Context) (uint64, bool) {
```

Comment:
how about moving this to Module.UnwrapStore() instead? then you don't need to do things like this for other features

Reply:
I am adding an UnwrapStore under Module and Instance.  It is more useful to me under Instance. Also added to wasmer as well as long as I was here anyway.

Reply:
sounds fine